wongnung/models.py

```diff
@@ -0,0 +1,136 @@
+    :type filmId: int
+    :param title: The title of a film.
+    :type title: str
+    :param year_released: The release year of the film.
+    :type year_released: int
+    :param director: The name of the director(s) of the film.
+    :type director: str
+    :param genres: The genres of the film.
+    :type genres: str
+    :param summary: A summary of the film.
+    :type summary: str
+    :param stars: The stars(actors and actresses) of the film
+    :type stars: str
+    """
+    filmId = models.IntegerField(primary_key=True)  # Required
+    title = models.CharField(max_length=256)  # Required
+    year_released = models.IntegerField()  # Required
+    director = models.CharField(max_length=512, blank=True)
+    genres = models.CharField(max_length=512, blank=True)
+    summary = models.CharField(max_length=1000, blank=True)
+    stars = models.CharField(max_length=1000, blank=True)  # names of
+
+    def get_director(self):
+        """Get a list contain the name of the director(s) of the film
+
+        :return: A list containing the string representing the name of the director(s) of the film
+        :rtype: list
+        """
+        return [director.strip() for director in
+                self.director.split(",")]
+
+    def get_genres(self):
+        """Get a list contain the genres of the film
+
+        :return: A list containing the string representing the genres of the film
+        :rtype: list
+        """
+        return [genre.strip() for genre in
+                self.genres.split(",")]
+
+    def get_stars(self):
+        """Get a list contain the genres of the film
+
+        :return: A list containing the string representing name of the stars of the film
+        :rtype: list
+        """
+        return [star.strip() for star in
+                self.stars.split(",")]
+
+    def set_director(self, directors: List[str]):
+        """Get a list containing the name of the director(s) of the film and set it to
+        be a director attribute
+
+        :param directors: A list containing the string representing the name of the director(s)
+        of the film
+        """
+        self.director = ", ".join(directors)
+
+    def set_genres(self, genres: List[str]):
+        """Get a list containing the genres and set it to be a genres attribute
+
+        :param genres: A list containing the string representing the genres of the film
+        """
+        self.genres = ", ".join(genres)
+
+    def set_stars(self, stars: List[str]):
+        """Get a list containing the stars of the film and set it to be a stars attribute
+
+        :param stars: A list containing the string representing the stars of the film
+        """
+        self.stars = ", ".join(stars)
+
+    @classmethod
+    def get_film(cls, film_id):
+        """Create and return Film object
+
+        :param film_id: The ID of a specific film.
+        :type film_id: int
+        :return: film object
+        :rtype: Film
+        """
```

Comment:
Use type hinting instead, also `film_id` shouldn't it be a `string`?

Reply:
Okay, I adjusted it. Thanks for pointing out